docs/sources/contribute-documentation/contribute-release-notes/index.md

```diff
@@ -1,73 +1,131 @@
 ---
-title: Contribute to What's new or release notes
-menuTitle: Contribute to What's new or release notes
-description: This section describes the different ways of contributing to the What's new document or release notes.
-weight: 250
 aliases:
   - /docs/writers-toolkit/writing-guide/contribute-release-notes
+date: 2024-02-14
+description: This section describes the different ways of contributing to the What's new document or release notes.
 keywords:
   - what's new
   - release notes
+menuTitle: Contribute to What's new or release notes
+title: Contribute to What's new or release notes
+weight: 250
 ---
 
+<!-- vale Grafana.Timeless = NO -->
+
 # Contribute to What's new or release notes
 
 This topic explains the decisions and actions associated with collecting, writing, and publishing What's new content.
 
-{{% admonition type="note" %}}
+{{< admonition type="note" >}}
 This topic is only relevant for internal Grafana Labs contributors.
-{{% /admonition %}}
+{{< /admonition >}}
 
-## What's new doc development process
+## What's new documentation development process
 
-What’s new content is published to the website through the website content management system (CMS), and can be accessed [here](https://admin.grafana.com/content-admin/#/collections/whats-new).
+You publish What’s new content to the website through the website Content Management System (CMS).
```

Comment:
While I get changing this out of passive voice, I don't think it improves the sentence or makes it easier to understand. And not using passive voice isn't a hard rule. Sometimes passive voice makes more sense and is easier to understand. I'd stick with the passive voice on this first sentence.
```suggestion
What’s new content is published to the website through the website Content Management System (CMS).
```